CHANGELOG.md

```diff
@@ -2,12 +2,27 @@
 
 All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org).
 
+## [v1.2.0](https://github.com/voxpupuli/puppet-puppetwebhook/tree/v1.2.0) (2019-03-01)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-puppetwebhook/compare/v1.1.0...v1.2.0)
+
+### Added
+
+- Add `pkg\_version` parameter [\#16](https://github.com/voxpupuli/puppet-puppetwebhook/pull/16) ([alexjfisher](https://github.com/alexjfisher))
+
+### UNCATEGORIZED PRS; GO LABEL THEM
```

Comment:
??? These have got our usual labels.  Just not the ones that this project's Rakefile has reconfigured.